AirCasting/SessionViews/EditView.swift

```diff
@@ -77,4 +82,10 @@ struct EditView<VM: EditViewModel>: View {
             presentationMode.wrappedValue.dismiss()
         }.buttonStyle(BlueTextButtonStyle())
     }
+    
+    private var customSpacer: some View {
+        Rectangle()
+            .fill(Color(UIColor.systemBackground))
+            .frame(height: 10)
+    }
```

Comment:
Why not just a padding instead?

Reply:
This is what happen when I added .`padding()`

<img width="308" alt="Screenshot 2022-06-28 at 12 57 15" src="https://user-images.githubusercontent.com/82096200/176162663-b18c92a1-88a7-4dc8-9ed8-9803eb211e0e.png">

Reply:
😂 I was going to say nothing

Reply:
@SzymonGorowski padding or spacer? 🤔

Reply:
@jankrzempek & @anna1901 thank you for digging 🤦  I realized that I put the `.padding()` in the wrong place.